### PR TITLE
#7 added footer

### LINKED ISSUE #7: Add a Footer
Add a nice footer to the app, with the links to all pages

### LINKED ISSUE #10: Add a pull request template, issue template and license
Exactly what it says

### PR DESCRIPTION
**Pull request by abhinavmalhotra01**
added a basic footer component

## Description
added a footer component in frontend

## Issue Resolved

Fixes #7

## Screenshots
Provided

## Checklist

Please make sure to review the following before submitting your PR:

- [✔️ ] I have read the [contribution guidelines](https://github.com/ken1000minus7/Bloggy/blob/master/README.md).
- [ ✔️] I have read the code of conduct.
- [✔️ ] I have reviewed my submission in detail.

### CODE REVIEW

**Comment by abhinavmalhotra01** [comment]
Screenshot 2023-02-05 133032


added the component to homepage

**Comment by abhinavmalhotra01** [comment]
sir do i need to add this to other pages also ?

**Comment by abhinavmalhotra01** [comment]
the icons and text are link tags to corresponding pages

File: src/frontend/src/pages/HomePage.js [hunk 1]
```diff
@@ -56,6 +57,7 @@ export const HomePage = ()=>{
                     latestBlogs && <BlogList blogList={latestBlogs.slice(0,Math.min(3,latestBlogs.length))} />
                 }
             </div>
+            <Footer/>
```
Comment (ken1000minus7) [inline src/frontend/src/pages/HomePage.js]: Add this in `App.js` not here, the footer should be added below the routes
https://github.com/ken1000minus7/Bloggy/blob/529e6922998239ac9f6f09bd6d8408b2fddb502b/src/frontend/src/App.js#L30-L42

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]
Remove the `.yarn-integrity` file, we don't need to commit any files in `node_modules`
Add the title and logo on the left of the footer and keep social media icons on the right
Also change the color to match the navbar

**Comment by abhinavmalhotra01** [comment]
Screenshot 2023-02-05 150248
sir kindly review this

**Comment by ken1000minus7** [comment]
Yeah, just increase the size of the title and logo and the social media icons and center them  vertically

**Comment by abhinavmalhotra01** [comment]
sir they are already vertically centered

**Comment by abhinavmalhotra01** [comment]
Screenshot 2023-02-05 153835


after increasing sizes

**Comment by ken1000minus7** [comment]
By vertically I meant, at the center of the height of the footer but chalo this looks fine as well

**Review by ken1000minus7 — APPROVED** [review]

**Comment by abhinavmalhotra01** [comment]
> By vertically I meant, at the center of the height of the footer but chalo this looks fine as well

oh sorry 😢

**Comment by abhinavmalhotra01** [comment]
pwoc site kind footer mtlb 🤔

**Comment by ken1000minus7** [comment]
Kinda

**Pull request merged**